torch/_torch_docs.py

```diff
@@ -4897,6 +4897,31 @@ def merge_dicts(*dicts):
     tensor([3, 2, 4])
 """.format(**common_args))
 
+add_docstr(torch.fmax, r"""
+fmax(input, other, *, out=None) -> Tensor
+
+Computes the element-wise maximum of :attr:`input` and :attr:`other`.
+
+.. note::
+    If one of the elements being compared is a NaN, then the non-nan element is returned. 
```

Comment:
"non-nan" -> "non-NaN"

Reply:
I think we can remove this note and just write the following:

Computes the element-wise maximum of :attr:\`input\` and :attr:\`other\`. This is like :func:\`torch.maximum\` except it handles NaNs differently: if exactly one of the two elements being compared is a NaN then the non-NaN element is taken as the maximum. Only if both elements are NaN is NaN propagated.

This function is a wrapper around C++'s \`\`std::fmax\`\` and is similar to NumPy's \`\`fmax\`\` function.

Supports :ref:`broadcasting to a common shape <broadcasting-semantics>`,
:ref:`type promotion <type-promotion-doc>`, and integer and floating-point inputs.

Reply:
Addressed!